tests/macsec/test_neighbor_reboot.py

```diff
@@ -0,0 +1,41 @@
+'''
+
+The test case will verify a neighbor reboot while Macsec is Active.
+
+Step 1: Configure Macsec between neighbor and DUT
+Step 2: Save config on neighbor and reboot
+Step 3: Verify macsec connection is re-established after reboot
+
+'''
+
+from time import sleep
+import pytest
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.macsec_required,
+    pytest.mark.topology("t2")
+]
+
+
+def test_neighbor_reboot(duthost, nbrhosts, request, enum_rand_one_frontend_asic_index):
```

Comment:
please use enum_frontend_dut_hostname to get duthost instead of directly using duthost fixture